libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewListener.kt

```diff
@@ -131,6 +131,15 @@ abstract class NavigationViewListener {
      */
     open fun onInfoPanelSettling() = Unit
 
+    /**
+     * Called when the info panel is being dragged.
+     *
+     * @param slideOffset The new offset of info panel within [-1,1] range. Offset increases
```

Comment:
I logged the values while enabling and disabling Info Panel in Free Drive and here is what I see:
1. When I enable the switch I receive 0, then I receive values from about -8 to 0, then I receive NaN. 
2. When I disable the switch I receive -1, then I receive values from 0 to -1.
So the values are not always within [-1, 1] range, and there are unexpected jumps.

Reply:
Blame BottomSheet implementation for this https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomsheet/BottomSheetBehavior.java#L109

Reply:
I believe the jumps are caused by `InfoPanelCoordinator.resetSlideOffset` function, so it is not really `BottomSheetBehavior` fault.

Reply:
Ahhh... Yeah... I see it now.
Technically need to only resetSlideOffset when InfoPanel is already expanded and the contentBinder changes. Let me see what I can do. Thx.

Reply:
Fixed. Please take a look when you have a moment.